www_src/pages/project/blocks/Link.jsx

```diff
@@ -0,0 +1,20 @@
+"use strict";
+
+var React = require("react");
+
+var Link = React.createClass({
+  getInitialState: function() {
+    return {
+      href: this.props.href || "",
+      label: this.props.label || "...",
```

Comment:
that seems the wrong reason to change this. It's the correct word for what it's used for, and since `<label>` elements only apply to form inputs, so the context doesn't really allow for that interpretation (and end users never see this value)